gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -247,6 +262,11 @@ public GoogleCloudStorageOptions build() {
       checkArgument(
           (instance.getProxyUsername() == null) == (instance.getProxyPassword() == null),
           "both proxyUsername and proxyPassword should be null or not null together");
+      checkArgument(
+          instance.getEncryptionAlgorithm() != null
```

Comment:
Should we check that either all of the encryption properties are set or neither of them are set?